resource pack.md

```diff
@@ -0,0 +1,34 @@
+# Serwerowy resource pack
+## Link do pobrania
+### Publicznego resource packa
+
+Żeby wgrać na serwer resource packa potrzebujemy bezpośredniego linku do pobrania go. Aby go zdobyć, należy kliknąć prawym przyciskiem myszyn na przycisk, 
+który spowoduje pobranie danej paczki zasobów. Następnie kopiujemy adres linku.
+![1](img/rscp/1.png)
+
+### Resource packa z komputera
+
+Jeśli chcemy wgrać na serwer własną paczkę zasobów, to trzeba przesłać ją na jakąś stronę, która umożliwia pobieranie jej innym osobom np. na [dropboxa](https://www.dropbox.com/). 
```

Comment:
```suggestion
Jeśli chcemy wgrać na serwer własną paczkę zasobów, to trzeba przesłać ją na jakąś stronę, która umożliwia pobieranie jej innym osobom np. na [Dropboxa](https://www.dropbox.com/). 
```

Warto tutaj zaznaczyć, że może to być dowolny serwis, ale musi on udostępniać możliwość bezpośredniego permalinku do pobrania.